tasks/service.yml

```diff
@@ -4,3 +4,15 @@
   ufw:
     state: "{{ ufw_state }}"
     logging: "{{ ufw_logging }}"
+
```

Comment:
I'm not sure if this is necessary... ufw itself has the state option and this controls if the service is running or not... did you encounter any issues?

Reply:
yes, ufw was not able on my system (Archlinux) to enable autostart and it didn't touch the systemd service state

Reply:
mmh, is there any doc on this one for archlinux cause adding this would be confusing to others. if the ufw command isn‘t working I‘ll have to check if it would be better to use the service call on all systems instead.